mods/arcade/web/style.css

```diff
@@ -308,7 +308,7 @@ body {
   font-weight: bold;
 }
 .gameShortDescriptionKey {
-  cloear: both;
+  clear: both;
```

Comment:
You might wnat to be careful doing things like this, analysis has historically not been done at the code level on saito-lite, if this has no effect the proper solution is probably to delete it.

Reply:
it's a typo for clearing float, I have never heard css cloear

Reply:
Yes, i know that.... my point is are you sure this class is supposed to have a clear?